packages/jsii-rosetta/lib/util.ts

```diff
@@ -102,6 +102,7 @@ export function mkDict<A extends string, B>(xs: Array<readonly [A, B]>): Record<
 
 export function fmap<A, B>(value: NonNullable<A>, fn: (x: A) => B): B;
 export function fmap<A, B>(value: undefined, fn: (x: A) => B): undefined;
+export function fmap<A, B>(value: A | undefined, fn: (x: A) => B): B | undefined;
 export function fmap<A, B>(value: A, fn: (x: A) => B): B | undefined {
```

Comment:
All of these direly lack doc-strings.

I have no idea what `fmap` means.
The body looks like this is a pretty trivial wrapper for `value != null ? fn(value) : undefined`... And having realized this I still don't see what `fmap` stands for?

Reply:
It's very much a wrapper for that. Just leads to more readable code than the tangle of `: undefined` you'd otherwise get into. 

I'll add docs.